azurerm/resource_arm_netapp_account.go

```diff
@@ -0,0 +1,288 @@
+package azurerm
+
+import (
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/Azure/azure-sdk-for-go/services/netapp/mgmt/2019-06-01/netapp"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/azure"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/response"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/tf"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/validate"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/features"
+	aznetapp "github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/services/netapp"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/tags"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func resourceArmNetAppAccount() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceArmNetAppAccountCreateUpdate,
+		Read:   resourceArmNetAppAccountRead,
+		Update: resourceArmNetAppAccountCreateUpdate,
+		Delete: resourceArmNetAppAccountDelete,
+
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ForceNew:     true,
+				ValidateFunc: aznetapp.ValidateNetAppAccountName,
+			},
+
+			"resource_group_name": azure.SchemaResourceGroupNameForDataSource(),
+
+			"location": azure.SchemaLocation(),
+
+			"active_directory": {
+				Type:     schema.TypeList,
+				Optional: true,
+				Elem: &schema.Resource{
```

Comment:
Are we sure there can be more then one of these?
```
------- Stdout: -------
=== RUN   TestAccAzureRMNetAppAccount_update
=== PAUSE TestAccAzureRMNetAppAccount_update
=== CONT  TestAccAzureRMNetAppAccount_update
--- FAIL: TestAccAzureRMNetAppAccount_update (83.58s)
    testing.go:569: Step 1 error: errors during apply:
        
        Error: Error creating NetApp Account "acctest-NetAppAccount-191008004136081477" (Resource Group "acctestRG-191008004136081477"): netapp.AccountsClient#CreateOrUpdate: Failure sending request: StatusCode=400 -- Original Error: Code="ErrorUpsertingActiveDirectory" Message="Only one active directory allowed"
        
          on /opt/teamcity-agent/temp/buildTmp/tf-test307630958/main.tf line 7:
          (source code not available)```

Reply:
After checked, it only supports one active directory. So I added attribute "MaxItems: 1".
![image](https://user-images.githubusercontent.com/19754191/66380832-bb916d80-e9ea-11e9-9844-9c72d7b291de.png)